File: array-easy/26.remove-duplicates-from-sorted-array.cs
/*
 * @lc app=leetcode id=26 lang=csharp
 *
 * [26] Remove Duplicates from Sorted Array
 */

// @lc code=start
public class Solution
{
    public int RemoveDuplicates(int[] nums)
    {
        var index = 0;
        var len = nums.Length;
        for (var i = 0; i < len; i++)
        {
            if (nums[i] == nums[index])
            {
                continue;
            }

            index++;
            nums[index] = nums[i];
        }

        return index + 1;
    }
}
// @lc code=end

File: array-easy/26.remove-duplicates-from-sorted-array.py
#
# @lc app=leetcode id=26 lang=python3
#
# [26] Remove Duplicates from Sorted Array
#

# @lc code=start
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        index = 0
        length = len(nums)

        for i in range(length):
            if nums[i] == nums[index]:
                continue

            index += 1
            nums[index] = nums[i]

        return index + 1


# @lc code=end
